src/AddTodoForm.tsx

```diff
@@ -1,9 +1,28 @@
-const AddTodoForm = () => (
-    <form id="addTodoForm">
-        <label htmlFor="todoTitle">Title</label>
-        <input id="todoTitle" />
-        <button value="Add">Add</button>
-    </form>
-);
-
-export default AddTodoForm;
+import { FC, FormEventHandler } from "react";
+
+interface AddTodoFormProps {
+	onAddTodo: (setNewTodo: string) => void;
+}
+
+const AddTodoForm: FC<AddTodoFormProps> = ({ onAddTodo }) => {
+	const handleAddTodo: FormEventHandler = (e) => {
+		e.preventDefault();
+		const input: HTMLInputElement | null =
+			e.currentTarget.querySelector("input");
+		if (input) {
```

Comment:
The input here is not conditionally rendered, so we do not need to (and should not) check if the input exists. There's a simpler approach to find the value of the input here.